Doc/includes/email-alternative.py

```diff
@@ -8,14 +8,14 @@
 
 # Create the base text message.
 msg = EmailMessage()
-msg['Subject'] = "Ayons asperges pour le déjeuner"
+msg['Subject'] = "Pourquoi pas des asperges pour ce midi ?"
 msg['From'] = Address("Pepé Le Pew", "pepe", "example.com")
 msg['To'] = (Address("Penelope Pussycat", "penelope", "example.com"),
              Address("Fabrette Pussycat", "fabrette", "example.com"))
 msg.set_content("""\
 Salut!
 
-Cela ressemble à un excellent recipie[1] déjeuner.
```

Comment:
A bit more wrong. The correct form would have been:
*Cela ressemble à une excellente recette pour le déjeuner*